signer/universal/universal_test.go

```diff
@@ -1 +1,41 @@
 package universal
+
+import (
+	"github.com/cloudflare/cfssl/config"
```

Comment:
style nit: this should be put below standard libarary and separated with one blank line.